### PR TITLE
fix(space-wizard): create space ux improvements fixes

### LINKED ISSUE #589: On https://prod-preview.openshift.io - selecting cancel after the new space wizard fails raises a new error
See attached screenshot - to recreate the error, create a new user space, select the wizard option, and then press cancel.

screenshot from 2017-04-09 22-21-16

### PR DESCRIPTION
**Pull request by mitchharpur**
closes #589

fixes some styling issues on create space and fixes close button

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: e57814a92b65907d3e937332338c7853f041158f
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@mitchharpur snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-812-fabric8-ui.int.rdu2c.fabric8.io

File: src/app/space-wizard/space-wizard.component.scss [hunk 1]
```diff
@@ -21,7 +21,7 @@ $wizard-layout-block-header-border-color: $color-pf-blue-300;
 //noinspection SassScssUnresolvedVariable,SassScssUnresolvedVariable
 $wizard-layout-block-shadow-color: darken($navbar-default-bg, 10%);
 //noinspection SassScssUnresolvedVariable,SassScssUnresolvedVariable
-$wizard-block-header-color: $navbar-default-bg;
+$wizard-block-header-color: white;// $navbar-default-bg;
```
Comment (joshuawilson) [inline src/app/space-wizard/space-wizard.component.scss]: Why isn't this $color-pf-white

Comment (mitchharpur) [inline src/app/space-wizard/space-wizard.component.scss]: experimenting ... not on that page yet ... will get to that

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 0fdaa38e3f78b918561ac3e0810dba66fb7e0cd5
  * Your subject line is longer than 50 characters

 * Commit: deca440345caae391260601c162585cafb1ba085
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 32db4d1a6a86dc422620ef02736b1b44cbb6aca8
  * Your subject line is longer than 50 characters

 * Commit: 5b96c8de9b3d39b59b40626604c77feffd4e5143
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 32db4d1a6a86dc422620ef02736b1b44cbb6aca8
  * Your subject line is longer than 50 characters

 * Commit: 5b96c8de9b3d39b59b40626604c77feffd4e5143
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by joshuawilson** [comment]
[test]

**Comment by fabric8cd** [comment]
@mitchharpur snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-812-fabric8-ui.int.rdu2c.fabric8.io

**Pull request merged**